components/articles/ArticlePreview.js

```diff
@@ -22,26 +23,26 @@ const ArticlePreview = ({
     <article className="card tile is-child is-flex">
       <div className="card-image">
         <figure className={classNames('image', imageClassName)}>
-          <ArticleLink slug={slug}>
+          <Link route={ROUTES_NAMES.article} params={{ slug }}>
             <a>
               <img alt={title} src={imageUrl || mockImagePath} />
             </a>
-          </ArticleLink>
+          </Link>
         </figure>
         {brand && <SpecialHeading {...brand} />}
       </div>
       <div className="card-content">
         <span className="title is-spaced">
-          <ArticleLink slug={slug}>
+          <Link route={ROUTES_NAMES.article} params={{ slug }}>
             <a>{title}</a>
-          </ArticleLink>
+          </Link>
         </span>
         <p className="subtitle">{subtitle}</p>
         {author && (
           <div className="author">
             <img alt={title} src={author.imageUrl || mockImagePath} />
             <div className="name">
-              {author.firstName} {author.lastName}
+              {author.firstName.be} {author.lastName.be}
```

Comment:
Will ``firstName``, ``lastName`` be always in belarusian?

Reply:
No, of course. It should be replaced with `displayName` after @UladBohdan api fix

Reply:
I see, thanks